ocs_ci/deployment/deployment.py

```diff
@@ -3444,6 +3449,105 @@ def deploy(self):
         else:
             self.enable_managed_serviceaccount()
 
+    @retry(ACMObservabilityNotEnabled, tries=10, delay=30)
+    def check_observability_status(self):
+        """
+        Check observability status
+        Returns (bool): True or False
+
+        """
+
+        acm_observability_status = bool(
+            run_cmd(
+                "oc get MultiClusterObservability observability -o jsonpath='{.status.conditions[1].status}'"
+            )
+        )
+
+        if acm_observability_status:
+            logger.info("ACM observability is successfully enabled")
+        else:
+            logger.error("ACM observability could not be enabled, re-trying...")
```

Comment:
Did you plan to add some code for re-trying?

Reply:
@retry decorator is already added to the function